java-frontend/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -700,6 +700,15 @@ private void buildBreakStatement(BreakStatementTree tree) {
       targetBlock = labelsBreakTarget.get(label.name());
     }
     currentBlock = createUnconditionalJump(tree, targetBlock);
+    if(currentBlock.exitBlock != null) {
+      currentBlock.exitBlock = null;
+      Block last = breakTargets.removeLast();
+      targetBlock.successors().clear();
```

Comment:
As dicussed, this does not require a fix in this PR. However shall this lead to open a ticket on the subject ?